docs/com/smartcar/sdk/AuthClient.html

```diff
@@ -376,15 +389,15 @@ <h4>AuthClient</h4>
                   java.lang.String&nbsp;clientSecret,
                   java.lang.String&nbsp;redirectUri,
                   java.lang.String[]&nbsp;scope,
-                  boolean&nbsp;development)</pre>
+                  boolean&nbsp;testMode)</pre>
 <div class="block">Initializes a new AuthClient.</div>
 <dl>
 <dt><span class="paramLabel">Parameters:</span></dt>
 <dd><code>clientId</code> - the application client ID</dd>
 <dd><code>clientSecret</code> - the application client secret</dd>
 <dd><code>redirectUri</code> - the registered redirect URI for the application</dd>
 <dd><code>scope</code> - the permission scope requested</dd>
-<dd><code>development</code> - whether or not to operate in development mode</dd>
+<dd><code>testMode</code> - whether or not mode should be set to 'test'</dd>
```

Comment:
thoughts on changing this to "Launch the Smartcar auth flow in test mode." to be consistent with the node and python SDKs?

Copy guideline :point_right:  https://smartcar.quip.com/CS0aAP8GEMEU#cPWACAmx71B